cpp/cmake/faiss_cuda11.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/faiss/gpu/utils/DeviceDefs.cuh b/faiss/gpu/utils/DeviceDefs.cuh
```

Comment:
Is this still needed now that we have the FAISS conda package?

Reply:
Yes, we'll need this change until we support BUILD_STATIC_FAISS flag in our build flow and/or FAISS folks enable sm_80 support officially.